src/modm/platform/can/windows_hosted/module.lb

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016-2018, Niklas Hauser
+# Copyright (c) 2017, Fabian Greif
+# Copyright (c) 2018, Christopher Durand
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+from collections import namedtuple
+from enum import Enum
+
+def load_options(module):
+    module.add_option(
+        NumericOption(
+            name="buffer.tx",
+            description="",
+            minimum=1, maximum=2 ** 16 - 2,
+            default=32))
+    module.add_option(
+        NumericOption(
+            name="buffer.rx",
+            description="",
+            minimum=1, maximum=2 ** 16 - 2,
+            default=32))
+
+CanType = Enum('CanType', 'Single Master Slave')
+CanInfo = namedtuple('CanInfo', 'register type associated_instance')
+
+can_map = {
+    "windows": {
+        1: CanInfo("1", CanType.Master, 2),
+        2: CanInfo("2", CanType.Slave , 1)
```

Comment:
What's the deal with this master/slave configuration?

Reply:
deleted this, wasn't actually using it